app/views.py

```diff
@@ -81,11 +81,33 @@ def progressreport():
     """Renders the report page."""
     return render_template('report.html', title=' | Progress Report')
 
-@app.route('/create', methods=['GET', 'POST'])
-def create():
-    """Renders the create page for educators."""
-    form = CreateQnForm()
-    return render_template('create.html', title=' | Create', form=form)
+@app.route('/createclass', methods=['GET', 'POST'])
+def createclass():
+    """Renders the create class page for educators."""
+    form = CreateClass()
+    if form.validate_on_submit():
+        Group.name = form.className.data
```

Comment:
That is not how you create a group.
```suggestion
        group = Group(name=form.className.data)
        db.session.add(group)
```